quickjs/build.gradle.kts

```diff
@@ -31,8 +31,15 @@ val versionWriterTaskProvider = tasks.register("writeVersion", VersionWriterTask
 
 val copyTestingJs = tasks.register<Copy>("copyTestingJs") {
   destinationDir = buildDir.resolve("generated/testingJs")
-  from(projectDir.resolve("testing/build/distributions/testing.js"))
-  dependsOn(":quickjs:testing:jsBrowserProductionWebpack")
+  if (true) {
+    // Production, which is minified JavaScript.
+    from(projectDir.resolve("testing/build/distributions/testing.js"))
+    dependsOn(":quickjs:testing:jsBrowserProductionWebpack")
```

Comment:
Do we ever actually care about this for tests?

Reply:
I’m undecided on whether I want tests to exercise production JS or development JS. Having development JS is really nice when things crash!